### PR TITLE
[HELM] feat: added missing monitoring options in values.yaml

### PR DESCRIPTION
**Pull request by dotdc**
Signed-off-by: David <davidcalvertfr@gmail.com>

### Checklist

- [x] Update changelog in CHANGELOG.md.
- [x] Make sure `values.yaml` and `values.schema.json` are valid.
- [ ] (Giant Swarm) If creating a release, bump the `version` and `appVersion` in Chart.yaml.

Hi,

Theses configuration options are present in the template files but not in the values.

Templates:
- [ServiceMonitor](https://github.com/giantswarm/starboard-exporter/blob/main/helm/starboard-exporter/templates/servicemonitor.yaml#L10)
- [GrafanaDashboard](https://github.com/giantswarm/starboard-exporter/blob/main/helm/starboard-exporter/templates/grafana-dashboard.yaml#L8)

It's easier for the users to have all the possible configuration options directly in the `values.yaml` file.

Updated `CHANGELOG.md`, let me know if anything's missing in my PR.

David

### CODE REVIEW

**Comment by dotdc** [comment]
CI is failing due to vulnerable dependencies, I don't think it's linked to my change but let me know.

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Spread (jitter) re-queueing of reports by +/- 10% by default to help smooth resource utilization.
+- Added missing monitoring options in the Helm chart values.yaml
```
Comment (stone-z) [inline CHANGELOG.md:15]: Can you please move this under a new `### Added` subheading under `[Unreleased]`? The release headings are automatically generated, so 0.4.1 refers to a release which is already out.

**Review by stone-z — COMMENTED** [review]
Hey @dotdc thanks for this. Looks fine to me, just one request for the changelog and we'll get this in.

File: CHANGELOG.md (hunk 1 above)
Comment (dotdc) [reply CHANGELOG.md]: My bad, should be good now :+1:

**Review by stone-z — APPROVED** [review]

**Comment by stone-z** [comment]
Thanks!

**Pull request merged**